movement/watch_faces/complication/totp_face_lfs.c

```diff
@@ -1,3 +1,27 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2022 James Haggerty (https://github.com/wryun)
```

Comment:
@wryun can you confirm this copyright?

Reply:
This is heavily based on @tahnok 's code, so I'd just use the notice from totp_face.c

Reply:
Acknowledged, I'm making the change now.